src/main/java/com/amazon/dlic/auth/http/saml/AuthTokenProcessorHandler.java

```diff
@@ -310,12 +312,12 @@ private String createJwt(SamlResponse samlResponse) throws Exception {
 
         if (token_log.isDebugEnabled()) {
             token_log.debug(
-                "Created JWT: "
-                    + encodedJwt
-                    + "\n"
-                    + jsonMapReaderWriter.toJson(jwt.getJwsHeaders())
-                    + "\n"
-                    + JwtUtils.claimsToJson(jwt.getClaims())
+                    "Created JWT: "
```

Comment:
nit: seems like unwanted indentation changes.

Reply:
Oops, this part was an accident, will fix asap